core/src/main/java/eu/fasten/core/data/FastenPythonURI.java

```diff
@@ -29,7 +31,7 @@ public class FastenPythonURI extends FastenURI {
     protected final boolean isFunction;
 
     public FastenPythonURI(final String s) {
-        this(URI.create(s));
+        this(URI.create(UrlEscapers.urlFragmentEscaper().escape(s)));
```

Comment:
From the [documentation](https://guava.dev/releases/19.0/api/docs/com/google/common/net/UrlEscapers.html) of the `urlFragmentEscaper()` I saw that it converts most of the special characters (including `< >`)  into one or more bytes using UTF-8 encoding. If the URI somehow includes a special character which is not allowed to exist in the URI, this conversion will not break the processing of the specific package version. Will this use case create issues further down the pipeline or e.g. in the analysies using the URI or it won't be a problem?

Reply:
I don't it should be an issue. The FASTEN URIs will be correctly encoded versions of the namespaces that PYCG produces into its call graphs. If downstream issues will emerge we should fix them there, but I doubt whether it will occur.

Reply:
Ok, I agree, let’s merge this!